webapp/src/components/training/TrainingSteps.vue

```diff
@@ -111,4 +114,207 @@ const unamedDataset = computed<Dataset<DataFormat.Raw[DataType]> | undefined>(
   },
 );
 const trainedModel = ref<Model<DataType>>();
+
+const guideStoreTraining = useGuideStore();
+
+//Create driver object
+const driverOBJ = driver({
+  showProgress: false,
+});
```

Comment:
I see that you're doing this pattern quite a lot, consider moving the driver creation and CSS import inside a dedicated helper, similar to `useToaster`